src-tauri/src/lib.rs

```diff
@@ -7,8 +9,15 @@ fn greet(name: &str) -> String {
 #[cfg_attr(mobile, tauri::mobile_entry_point)]
 pub fn run() {
     tauri::Builder::default()
+        .plugin(tauri_plugin_os::init())
+        .on_window_event(|_, event| {
+            if let WindowEvent::Resized(_) = event {
+                std::thread::sleep(std::time::Duration::from_nanos(1));
+            }
+        })
         .plugin(tauri_plugin_shell::init())
         .invoke_handler(tauri::generate_handler![greet])
-        .run(tauri::generate_context!())
-        .expect("error while running tauri application");
+        .build(tauri::generate_context!())
+        .expect("error while running tauri application")
+        .run(|handle, evt| {})
 }
```

Comment:
handle and evt are unused

```suggestion
.run(|_handle, _evt| {})
```